modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a private storage endpoint on Azure with user-provided VNet and subnet names.
+
+.Prerequisites 
+
+* You have configured the Image Registry Operator for use with Azure.
+
+.Procedure 
+
+. Enter the following command to obtain the VNet and subnet names that are used in the cluster:
++
+[source,terminal]
+----
+$ oc get machinesets -nopenshift-machine-api -ojson|jq '.items[1].spec.template.spec.providerSpec.value | (.vnet, .subnet)'
+----
+
+. Edit the Image Registry Operator `config/instance` object and configure the private endpoint using your VNet and subnet names:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry/cluster
+----
++
+[source,terminal]
+----
+# ...
+spec:
+  [...]
+   storage:
+      azure:
+        [...]
+        networkAccess:
+          type: Internal
+          internal:
+            subnetName: <subnet_name>
+            vnetName: <vnet_name>
+# ...
+----
+
+. Wait until the Image Registry Operator configures the private endpoint. Enter the following command to return the endpoint name:
```

Comment:
I had this as a testing step because it helped me know when the operator was done configuring the private endpoint. End users don't need to know the private endpoint name.
Maybe we make this an optional step? Maybe something like "(Optional) Watch the private endpoint name to know when the operator completed provisioning - this may take a few minutes"?

Reply:
Made it optional in both procedures.